andela/ah-technocrats-frontend
# #161966567 Signup New Users

### What does this pull request do?
 - This pull request creates the Signup Page. It allows a user to create an account for Authors Haven.
using their email and username.
### Description of the tasks to be completed?
 - Creates a sign up page with validation.
### How should this be manually tested?
 - clone the repo `https://github.com/andela/ah-technocrats-frontend.git`
 - change to the cloned directory `cd ah-technocrats-frontend`
 - `git checkout ft-registration-161966567` to checkout to this branch
 - run `npm install` to install the application.
 - run npm start

- To log in, Enter your username and email and a password.
- The form will validate the entries according to the data provided. The following are the validations present:
    - Usernames should be at least five characters
    - Passwords should have at least eight characters, one special character and one capital letter.
    -  None of the fields may be blank.

### Screenshots
  The form displaying all the errors
<img width="742" alt="screen shot 2019-01-18 at 09 35 52" src="https://user-images.githubusercontent.com/25079238/51369407-86fc1f80-1b04-11e9-902b-5a3ca7caab55.png">

- The Form after a successful login.
<img width="1061" alt="screen shot 2019-01-18 at 09 59 53" src="https://user-images.githubusercontent.com/25079238/51370332-d8f27480-1b07-11e9-9a3d-fdd27596d359.png">






### What are the relevant Pivotal Tracker Stories
[#161966567](https://www.pivotaltracker.com/story/show/161966565)
# Checklist:
